py/examples/text_alignment.py

```diff
@@ -0,0 +1,33 @@
+# Form / Text / Alignment
```

Comment:
Please add a screenshot of this example to `website/docs/examples/assets` to make docs build pass.

Reply:
@mturoci Thanks for letting me know about committing a screen shot for the new example. Here's a screen shot of the page after I rebuilt all the docs:

![doc_example_issue_1370_20240523](https://github.com/h2oai/wave/assets/61979454/21fabc1c-6f73-41d8-9c63-aeecd409db15)